product_pricelist_alternative/__manifest__.py

```diff
@@ -3,14 +3,15 @@
 
 {
     "name": "Product Pricelist Alternative",
-    "version": "18.0.1.0.0",
+    "version": "18.0.1.1.0",
     "development_status": "Beta",
     "category": "Product",
     "summary": "Calculate product price based on alternative pricelists",
     "author": "Camptocamp, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/product-attribute",
     "license": "AGPL-3",
     "depends": [
+        "sale",
```

Comment:
IMHO, this is not correct. The module should not depends on sale for such changes.

You should do a glue module.

Reply:
16.0 version depends of product
https://github.com/OCA/product-attribute/blob/ec24dffa276cbbf92a316e110a7c929c736e2809/product_pricelist_alternative/__manifest__.py#L14

Reply:
I'll do a glue module `sale_product_pricelist_alternative` and submit it to OCA/sale-workflow, do we agree?

Reply:
or the glue module should be added to OCA/product-attribute? :thinking: